totalRP3/core/impl/communication_protocol_broadcast.lua

```diff
@@ -320,20 +361,27 @@ Comm.broadcast.init = function()
 			-- We'll send out the event nice and early to say we're setting up.
 			TRP3_API.events.fireEvent(TRP3_API.events.BROADCAST_CHANNEL_CONNECTING);
 
+			-- Allow to join the broadcast channel even if the channel list is not ready
+			-- after 10 seconds.
+			local canJoinWhenNotReady = false;
+			C_Timer.After(10, function() canJoinWhenNotReady = true end);
```

Comment:
`canJoinWhenNotReady` is a bit of an unclear name.

```suggestion
			-- Force joining the broadcast channel if we wait too long.
			local forceJoinChannel = false;
			C_Timer.After(10, function() forceJoinChannel = true; end);
```